libs/fabric/src/lib/components/combo-box/directives/combo-box-option.directive.ts

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { Directive, Input } from '@angular/core';
+import { IComboBoxOption } from 'office-ui-fabric-react';
+
+/**
+ * Wrapper directive for creating a ComboBoxOption
+ */
+@Directive({ selector: 'fab-combo-box-option' })
+export class ComboBoxOptionDirective {
+  @Input() optionKey: IComboBoxOption['key'];
```

Comment:
Why not call it `key` in the property name too? We'd like to stay consistant with the `office-ui-fabric-react` names as much as possible, especially with stuff like #91   in the pipeline.

Reply:
I concur, but that runs foul of the wrapper-component _isForbiddenAttribute method. This causes the ComboBox to misbehave (it ignores the key attribute binding and triggers a console warning).

Reply:
Makes sense, let's keep it like that then. Please do add some JDoc on this to make a note of this.